### ISSUE TITLE
DataCite Roles may not be a good match for DMP roles?

### ISSUE THREAD

**Issue by rwwh** [RDA-DMP-Common/RDA-DMP-Common-Standard#16]
In the model, the contributor is a contributor to the DMP, not to the project, but the contributor.role vocabulary from datacite is defining "contributors" to a "resource": I interpret that as roles for a *product* of a project, not of the project or the DMP. The DMP /could/ be seen as a product(?)

Further: I have a hard time to identify e.g. a suitable role for the "data steward" in a project, e.g. the leader of a work package "data management" in a research project, "policing" the implementation of the data management plan. I now suggest "Data Curator" in the [DS-Wizard](https://ds-wizard.org), but it is a stretch of the definition.

Also: The allowed contributorTypes includes some "institutes" or "groups" rather than people. Groups and institutes are identified by other identifiers, certainly not ORCID.

**Comment by TomMiksa** [comment]
I think the best solution for us in this case is to drop the datacite dictionary. It can only lead to inconsistencies. I know also about this list of roles from the LOC:
https://www.loc.gov/marc/relators/relaterm.html But it also does not solve the problem

Basically, there is no single dictionary that would cover:
-	Roles of people involved in managing data
-	Contributors to the DMP
Splitting these into two concepts would not really help either. 

I think we should make the following changes:
Contributor/role -> Data type set to String instead of Controlled Vocabulary
Contributor/role -> Cardinality set to 1..* so that one person could have multiple roles

Then you will be able to say that Rob Hooft is a Manager and a Data Steward and you won’t be constrained by datacite. 

BTW, Now I see that the future work for RDA is really in defining the dictionaries. I don’t see though much enthusiasm to do it within the community.

**Comment by paulwalk** [comment]
I agree that this is far from clear enough. I agree with Tomasz'z suggestion to drop the reference to DataCite for this property, and to change this to a string value. I will make this change now and close this issue. (I have already made the change to the cardinality of roles).

**Comment by rwwh** [comment]
Agreed, but I hope this does not cut off the route towards a new controlled vocabulary later.

**Comment by paulwalk** [comment]
OK - I should not have closed the issue so have reopened it. I meant that we have made a short-term decision - but there is certainly a long-term issue to resolve!

**Comment by briri** [comment]
+1 for changing the cardinality of the contributor/role to 1..*

Has there been a consensus on which ontology can be used to best describe contributor roles for a DMP?

We will be supporting multiple roles per contributor in the DMPRoadmap data model. We are planning on using the CRediT taxonomy (for now at least) https://casrai.org/credit/.
In particular: 
- the 'Data curation' role for the 'contact' of the DMP (although this feels like a bit of a stretch)
- 'Writing - original draft', 'Writing - review & editing' for authors, editors and reviewers of the DMP - - 'Investigation' to identify principal investigator(s)

**Comment by briri** [comment]
just realized that role is an array in the new schema v.1. Will switch to use the 
``json
"contributors": [
    { 
      "name": "Jane Doe", 
      "mbox": "jane.doe@example.org", 
      "roles": [ 
        "https://dictionary.casrai.org/Contributor_Roles/Investigation", 
        "https://dictionary.casrai.org/Contributor_Roles/Writing_original_draft"
      ] 
    }
  ]
```

**Comment by mabablue** [comment]
> I think the best solution for us in this case is to drop the datacite dictionary. It can only lead to inconsistencies. I know also about this list of roles from the LOC: https://www.loc.gov/marc/relators/relaterm.html But it also does not solve the problem
> 
> Basically, there is no single dictionary that would cover:
> 
> * Roles of people involved in managing data
> * Contributors to the DMP
>   Splitting these into two concepts would not really help either.
> 
> I think we should make the following changes: Contributor/role -> Data type set to String instead of Controlled Vocabulary Contributor/role -> Cardinality set to 1..* so that one person could have multiple roles
> 
> Then you will be able to say that Rob Hooft is a Manager and a Data Steward and you won’t be constrained by datacite.
> 
> BTW, Now I see that the future work for RDA is really in defining the dictionaries. I don’t see though much enthusiasm to do it within the community.

Hi all, I see that you suggest to use the  Contributor-Role-Ontology (https://github.com/data2health/contributor-role-ontology) but the property is still set to datatype; you should change it back to object property to accept roles from a controlled vocabulary. Is there any chance you will do this soon?

**Comment by TomMiksa** [comment]
Hi Barbara,

are you referring to make changes in this ontology: https://github.com/RDA-DMP-Common/RDA-DMP-Common-Standard/tree/master/ontologies

Or you mean changes in the recommendation in general?

Cheers
Tomasz

**Comment by mabablue** [comment]
you are right I am referring to the ontology. Sorry for my late reply.

**Comment by MarekSuchanek** [comment]
So, the issue is currently we have `Contributor.role` as a `string`. Limiting it to specific vocabulary would lead into breaking change. What we can do now for v1.2 is to suggest use of certain vocabulary/vocabularies.

I tried to take a look at following and really not sure if any of those is perfectly suitable as we want *Contributors role(s) within the process of data management (incl. planning)* (coming from #110):

- [Contributor Role Taxonomy (CRediT)](https://credit.niso.org/) = 14 roles can be related quite well to data management, but there are also "writing" and there it may not be clear if that is writing of DMP or some publications related to the project, and similarly "project administration" could cover also putting together a DMP; we could make a table of the 14 roles and relate it to data management to make it clear when to use which (as @briri started)
- [Contributor-Role Ontology](https://github.com/data2health/contributor-role-ontology) = is really designed for "data4health" (they should probably reflect that in the ontology name), it contains stuff like "patient advocate role", "standards developer role" and other that are not suitable for our work here
- [MARC Code List for Relators (LOC.gov)](https://www.loc.gov/marc/relators/relaterm.html) = way too broad, many that are not suitable for data management but we may suggest some subset...
- [DataCite Contributor Type](https://datacite-metadata-schema.readthedocs.io/en/4.5/appendices/appendix-1/contributorType/) = I think this one is the closest to what we want, we already use parts of DataCite... but those are more oriented towards project management, this is also used/extended in [OpenAIRE contributor type](https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/latest/vocab_contributortype.html)
- [DDI Alliance Controlled Vocabulary for Contributor Role](https://vocabularies.cessda.eu/vocabulary/ContributorRole) = relatively close to DataCite but extended with more roles

Questions:

- Would be a way to go to suggest *DataCite Contributor Type* (in some unified string format) for v1.2 and eventually change the type for v2?
- Any objections or counter proposals?

**Comment by briri** [comment]
Yes, I think leaving it as a string and making the suggestion to use _Datacite Contributor Type_ is a good first step for v1.2.

**Comment by mabablue** [comment]
so... @briri, @MarekSuchanek does that mean... that we can and should use PIDs for a datatype property? When do you plan to make a version 2?
